flybywiresim/aircraft
# fix(fm): don't update T-P position if we aren't inserting a direct to

Fixes #8968

## Summary of Changes

- Don't update T-P position if we aren't inserting a direct to
- Make T-P position update properly edit the definition so that it works with sync

## Testing instructions


- Make sure T-P updates when inserting a direct to
  - Wait a minute until confirming a direct to to a far away waypoint - make sure the T-P point moves to the airplane position when inserting
 - Make sure T-P does not move when it is the FROM leg and making another modification that creates a temporary plan, for example, adding a hold somewhere
   - Confirm the direct to, then make the unrelated modification a few moments after (so you can notice whether the T-P moves or not)


## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo** or **flybywire-aircraft-a380-842** download link at the bottom of the page
